pr-data.csv

```diff
@@ -1859,9 +1859,9 @@ https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c
 https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.JsonDeserializationTest.testWithConstructors,ID,,,
 https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.JsonUnwrappedTest.test,ID,Opened,https://github.com/vojtechhabarta/typescript-generator/pull/688,
 https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.JsonUnwrappedTest.testPrivateField,ID,Opened,https://github.com/vojtechhabarta/typescript-generator/pull/688,
-https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.MapEntryTest.testDefaultShapes,ID,,,
-https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.MapEntryTest.testListOfMapEntry,ID,,,
-https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.MapEntryTest.testOverriddenShapes,ID,,,
+https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.MapEntryTest.testDefaultShapes,ID,Opened,https://github.com/vojtechhabarta/typescript-generator/pull/717,
+https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.MapEntryTest.testListOfMapEntry,ID,Opened,,
```

Comment:
Missing PR link for two lines here.

Reply:
I have added the PR link.

Reply:
Sorry I didn't realize that PR is only for the first test and not the other two. Please remove `Opened` from these two for now. Hopefully the 717 PR will be accepted, so you can open another for the other two tests.

Reply:
I just removed the opened label and moved the link to the note column.